content/inlmiscmaps.md

```diff
@@ -144,7 +144,7 @@ cat(sprintf("width = %s, height = %s", din[1], din[2]))
 Replotting the map using the returned device dimensions results in
 a figure that is void of extraneous white space.
 
-<img src='/static/inlmiscmaps/plot_volcano_din-1.png'/ title='Static map of valcano data set with improved device dimensions.' alt='Static map of valcano data set with improved device dimensions.' class=''/>
+{{< figure src="/static/inlmiscmaps/plot_volcano_din-1.png" title="Static map of valcano data set with improved device dimensions." alt="Static map of valcano data set with improved device dimensions." >}}
```

Comment:
This is probably my ignorance but wanted to ask anyway: do these `figure` tags need the `/` to close the tag? So for instance on line 147, rather than `>}}` should it be `/>}}`?

Reply:
So these are shortcodes and are a Hugo thing. See https://gohugo.io/content-management/shortcodes/#figure